routers/api/packages/chef/chef.go

```diff
@@ -393,7 +394,7 @@ func DeletePackage(ctx *context.Context) {
 	}
 
 	for _, pv := range pvs {
-		if err := packages_service.RemovePackageVersion(ctx, ctx.Doer, pv); err != nil {
+		if err := common.RemovePackageVersion(ctx, ctx.Doer, pv); err != nil {
```

Comment:
It calls `RemovePackageVersion` mutiple times and "build" again and again. It doesn't seem right.

It should introduce a final step to rebuild when the removal finishes.